web-common/src/features/models/workspace/ModelBody.svelte

```diff
@@ -44,6 +45,8 @@
   export let modelName: string;
   export let focusEditorOnMount = false;
 
+  let tableQuery;
```

Comment:
This is redundant. The reactive statement `$: tableQuery = ..` will initialize the variable for us.

Reply:
Done